server/authn/presharedkey/presharedkey.go

```diff
@@ -5,10 +5,14 @@ import (
 
 	"github.com/go-errors/errors"
 	grpcAuth "github.com/grpc-ecosystem/go-grpc-middleware/auth"
-
 	"github.com/openfga/openfga/server/authn"
+	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
+var errUnauthenticated = status.Error(codes.Code(openfgapb.AuthErrorCode_unauthenticated), "unauthenticated")
```

Comment:
This should move up to `authn/authn.go` as well. It's generic enough that it belongs there. Sorry, that's what I meant before, I just wasn't clear.

Reply:
Got it. Updated. Thanks!